plaso/multi_processing/psort.py

```diff
@@ -84,13 +84,22 @@ def _GetEventIdentifiers(self, event, event_data, event_data_stream):
       event_attributes.extend(event_data_stream.GetAttributes())
 
     for attribute_name, attribute_value in sorted(event_attributes):
-      if attribute_name in self._IDENTIFIER_EXCLUDED_ATTRIBUTES:
+      # The filestat parser operated on file entry level and has no event data
+      # stream with a path specification. Therefore we need filename and inode
+      # to make events regarding different file entries are considered unique.
```

Comment:
I think is supposed to read "...to make sure events..."

Reply:
done